.travis.yml

```diff
@@ -19,8 +19,7 @@ install:
   - cd ..
 
 script:
-  # NOTE: c not built -- fix later
-  - make clean json franca binary csv deploy
+  - make travis_targets
```

Comment:
Later we can change also .travis.yml in vss-tools. when that is done we only need to change on file (makefile in vss) to change which targets to build for travis tests for both vss and vss-tools